musicbot/bot.py

```diff
@@ -1950,6 +1950,27 @@ def check(m):
                 raise exceptions.CommandError(
                     'Unreasonable volume provided: {}%. Provide a value between 1 and 100.'.format(new_volume), expire_in=20)
 
+    async def cmd_autoplaylist(self, channel, author, player, voice_channel):
+        """
+        Usage:
+            {command_prefix}autoplaylist
+			
+		Enable or disable the autoplaylist
+        """
+        song_url = random.choice(self.autoplaylist)       
+        if self.config.auto_playlist == False:
+            self.config.auto_playlist = True
+            if not player.playlist.entries and not player.current_entry and self.config.auto_playlist: #if nothing is queued, start a song
+                await player.playlist.add_entry(song_url, channel=None, author=None)
+            await self.safe_send_message(channel, "The autoplaylist is now enabled!")
```

Comment:
This should return a `Response` object instead, along with the same line in the section below.